pkg/cmd/dedicated/register/registercluster.go

```diff
@@ -59,6 +58,8 @@ func NewRegisterClusterCommand(f *factory.Factory) *cobra.Command {
 	}
 
 	// TODO add Localizer and flags
+	// add a flag for clustermgmt url, i.e --cluster-management-api-url, make the flag hidden, api.openshift.com or supply
+	// supply customer mgmt access token via a flag, i.e --access-token, make the flag hidden
```

Comment:
```suggestion
	// add a flag for clustermgmt url, i.e --cluster-management-api-url, make the flag hidden, default to api.openshift.com 
	// supply customer mgmt access token via a flag, i.e --access-token, make the flag hidden, default to ""
```

Reply:
Just a note:

The flags are only temporary for testing until there is a metadata API from KFM

When creating a conncection to OCM, we use the provided api url. And optionally use the access token if it is not ""